docs/howto/request-public-listing.md

```diff
@@ -0,0 +1,78 @@
+(make-your-charm-discoverable)=
+# Make your charm discoverable
+
+> See first: [Charm maturity](#charm-maturity)
+
+Once your charm is ready for wide production use, get it publicly listed on [Charmhub](https://charmhub.io), so that it is visible in searches done by Juju users and other charm developers.
```

Comment:
```suggestion
Once your charm is ready for wide production use, your next goal should be to get it publicly listed on [Charmhub](https://charmhub.io), so that it is visible in searches done by Juju users and other charm developers.
```